src/components/httproute/filters/FilterActions.tsx

```
@@ -0,0 +1,788 @@
+                            </FormGroup>
+                            <FormGroup label={t('Port')} fieldId="rr-port">
+                              <TextInput
+                                id="rr-port"
+                                type="number"
+                                value={
+                                  (f.requestRedirect?.port as number | undefined)?.toString?.() ||
+                                  ''
+                                }
+                                onChange={(_, value) => {
+                                  const parsed = Number(value);
+                                  const portNum = value && Number.isFinite(parsed) ? parsed : undefined;
```

ESLint

[error] Insert `⏎···································`

(prettier/prettier)